[DOCS-10667] Update RUM <> APM image and add nav steps (#28931)
DataDog/documentation: content/en/real_user_monitoring/correlate_with_other_telemetry/apm/_index.md

{{% /tab %}}
{{< /tabs >}}

## RUM Explorer to Traces

{{< img src="real_user_monitoring/connect_rum_and_traces/rum-trace-apm-link.png" alt="RUM and Traces" style="width:100%;">}}

To view traces from the RUM Explorer:

> 1. In the RUM Explorer, navigate to your [list of sessions][21] and click on a session with traces available (you can query for `@_dd.trace_id:*`).
> 1. The session panel appears with a request duration breakdown and includes a flame graph of each span, as well as a link to **View Trace in APM**.

Reviewer: The second point isn't really a step/action, so I think it makes more sense to pull it out of the step list? 

I also removed "In the RUM Explorer" since the preceding line captures that meaning, imo. 

```suggestion
1. Navigate to your [list of sessions][21] and click on a session that has traces available. You can also query for sessions with traces by using`@_dd.trace_id:*`.

When you select a session, the session panel appears with a request duration breakdown, a flame graph for each span, and a **View Trace in APM** link.
```

## Traces to RUM Explorer

{{< img src="real_user_monitoring/connect_rum_and_traces/rum-traces-to-rum.png" alt="RUM and Traces" style="width:100%;">}}

To view the RUM event from Traces:

> 1. Within a trace view, click **VIEW** to sell all traces created during the view life span, or **RESOURCE** to see traces associated with the specific resource from the Overview tab view.
> 1. Click **See View in RUM** or **See Resource in RUM**. You are taken to the view with the trace(s).

Reviewer: ```suggestion
1. Within a trace view, click **VIEW** to see all traces created during the view’s lifespan, or **RESOURCE** to see traces associated with the specific resource from the Overview tab.
1. Click **See View in RUM** or **See Resource in RUM** to open the corresponding event in the RUM Explorer.
```

These HTTP headers are not CORS-safelisted, so you need to [configure Access-Control-Allow-Headers][17] on your server handling requests that the SDK is set up to monitor. The server must also accept [preflight requests][18] (OPTIONS requests), which are made by the browser prior to every request when tracing is allowed on cross-site URLs.

> ## Affect on APM quotas

Reviewer: ```suggestion
## Effect on APM quotas
```

Connecting RUM and traces may significantly increase the APM-ingested volumes. Use the initialization parameter `traceSampleRate` to keep a share of the backend traces starting from browser and mobile requests.

> ## Traces retention

Reviewer: ```suggestion
## Trace retention
```
